chirp-impl/src/main/java/sample/chirper/chirp/impl/ChirpTimelineEntity.java

```diff
@@ -0,0 +1,34 @@
+package sample.chirper.chirp.impl;
+
+import akka.Done;
+import akka.NotUsed;
+import com.lightbend.lagom.javadsl.persistence.PersistentEntity;
+import sample.chirper.chirp.impl.ChirpTimelineCommand.AddChirp;
+import sample.chirper.chirp.impl.ChirpTimelineEvent.ChirpAdded;
+
+import javax.inject.Inject;
+import java.util.Optional;
+
+class ChirpTimelineEntity extends PersistentEntity<ChirpTimelineCommand, ChirpTimelineEvent, NotUsed> {
+    private final ChirpTopic topic;
+
+    @Inject
+    ChirpTimelineEntity(ChirpTopic topic) {
+        this.topic = topic;
+    }
+
+    @Override
+    public Behavior initialBehavior(Optional<NotUsed> snapshotState) {
+        BehaviorBuilder b = newBehaviorBuilder(NotUsed.getInstance());
+        b.setCommandHandler(AddChirp.class, this::addChirp);
+        b.setEventHandler(ChirpAdded.class, evt -> state());
+        return b.build();
```

Comment:
BTW I get a bunch of unchecked assignment warnings from IntelliJ in this method and also on the `ctx.reply` call below.

- Unchecked assignment: 'com.lightbend.lagom.javadsl.persistence.PersistentEntity.BehaviorBuilder' to 'com.lightbend.lagom.javadsl.persistence.PersistentEntity.BehaviorBuilder'
- Unchecked assignment: 'com.lightbend.lagom.javadsl.persistence.PersistentEntity.Behavior' to 'com.lightbend.lagom.javadsl.persistence.PersistentEntity.Behavior'
- Unchecked call to 'reply(R)' as a member of raw type 'com.lightbend.lagom.javadsl.persistence.PersistentEntity.ReadOnlyCommandContext'

I'm not totally sure what's wrong... possibly because those inner classes inherit a type parameter from the enclosing class?